libraries/Microsoft.Bot.Builder.Skills/SkillHandler.cs

```diff
@@ -82,7 +82,7 @@ public class SkillHandler : ChannelServiceHandler
         /// <returns>task for a resource response.</returns>
         protected override Task<ResourceResponse> OnSendToConversationAsync(ClaimsIdentity claimsIdentity, string conversationId, Activity activity, CancellationToken cancellationToken = default)
         {
-            return InvokeChannelApiAsync<ResourceResponse>(_adapter, _bot, claimsIdentity, ChannelApiMethods.SendToConversation, conversationId, activity);
+            return HandleChannelRequestAsync<ResourceResponse>(claimsIdentity, ChannelApiMethods.SendToConversation, conversationId, activity);
```

Comment:
Is this method used for handling a request from a skill? Should it be `HandleSkillRequestAsync`?

If I understand the flow:
1. Caller forwards message to skill, gets a in-proc response 
    - _(Random aside: are Skills a scenario where 202 is the primary valid 2xx status code?)_
2. Any out-of-proc requests from the skill, they hit the caller which has a SkillController of some sort implemented
3. This controller maps to the `SkillHandler`, which performs the requested operation on the skill's behalf and then sends the response back to the skill

Reply:
I am refactoring this class and will rename the methods so they are more understandable (these classes and methods are internal or private so it is just for our own benefit to make maintenance easier, the name is not visible to bot devs).

Your understanding of the sequence is correct.